src/base/emu-i386/simx86/interp.c

```diff
@@ -481,18 +481,36 @@ static void HandleEmuSignals(void)
 		CEmuStat &= ~(CeS_SIGPEND | CeS_RPIC);
 }
 
-static unsigned int _Interp86(unsigned int PC);
+static TNode *_Interp86(unsigned int PC, unsigned int Interp_LONG_CS,
+			int basemode, int gap);
+static unsigned int interp_pre(unsigned int PC);
 static unsigned int InterpOne(unsigned int PC, unsigned int Interp_LONG_CS,
 			      int basemode);
 
 void Interp86(void)
 {
-    unsigned int ret;
+    unsigned int PC;
 
     TheCPU.err = 0;
-    ret = _Interp86(LONG_CS + TheCPU.eip);
+
+    if (PROTMODE() && setjmp(jmp_env)) {
+      /* long jump to here from simulated page fault
+	 via Gen_sim or Sim_helper, with different cs:eip */
```

Comment:
Something with indent?

Reply:
you can change it in https://github.com/settings/appearance